test/elixir_conf_africa/events_test.exs

```diff
@@ -22,11 +24,38 @@ defmodule ElixirConfAfrica.EventsTest do
       assert Events.list_events() == [event]
     end
 
+    test "get_elixir_conf_event /0 returns the elixir conf event" do
+      event =
+        insert!(:elixir_conf_event)
+
+      assert Events.get_elixir_conf_event().name == event.name
+    end
+
     test "get_event!/1 returns the event with given id" do
       event = event_fixture()
       assert Events.get_event!(event.id) == event
     end
 
+    test "get_elixir_conf_event_and_ticket_types/0 returns the elixir conf event with ticket types" do
+      event =
+        insert!(:elixir_conf_event)
+
+      ticket_type =
+        insert!(:elixir_conf_ticket_type, event_id: event.id)
+
+      assert Events.get_elixir_conf_event_and_ticket_types().ticket_types != []
+    end
+
+    test "get_all_available_tickets/0 returns the number of available tickets" do
+      event =
+        insert!(:elixir_conf_event)
+
+      ticket_type =
+        insert!(:elixir_conf_ticket_type, event_id: event.id)
+
+      assert Events.get_all_available_tickets() == ticket_type.number
```

Comment:
Maybe rename this function to `get_number_of_tickets_available` with the current it suggests maybe we are getting a list of tickets.

also believe we should give this function parameter to make this more dynamic.

```  def get_all_available_tickets do
    query =
      from t in TicketType,
        join: e in Event,
        on: t.event_id == e.id and e.name == "ElixirConf Africa 2024",
        select: sum(t.number)

    Repo.one(query)
  end
  ```
  But if we want a specific event name say in this case `ElixirConf Africa 2024` then
  the name should be `get_number_of_tickets_available_for_elixir_conf_2024`(long 😄  but explicit)

Reply:
I agree , I have changed the function name to  `get_number_of_tickets_available_for_elixir_conf_2024` and added documentation for the same